lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php

```diff
@@ -36,34 +36,28 @@ public function convertException($message, DriverException $exception)
                 return new Exception\LockWaitTimeoutException($message, $exception);
             case '1050':
                 return new Exception\TableExistsException($message, $exception);
-
```

Comment:
This is an undesired change. See https://github.com/doctrine/coding-standard/pull/169.

Reply:
The rule that requires this change is `SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing.IncorrectLinesCountAfterLastControlStructure`

Reply:
Oh I get what you mean now. I should ignore it until 8.0.0

Reply:
Done.